plugin.audio.pureradio/resources/lib/api.py

```diff
@@ -0,0 +1,14 @@
+# _*_ coding: utf-8 _*_
+
+import urllib.request as request
+from xml.etree import ElementTree as ET
+
+# functions with web interface
+def retrieve_streamurl():
+    return """http://pureradio.one:8000/"""
+
+def retrieve_podcasturl():
+    return """http://www.spreaker.com/show/1757189/episodes/feed"""
```

Comment:
Are you sure this what you want here?

Reply:
replace the static string with response from web server.